### ISSUE TITLE
System card viewer in frontend

### ISSUE THREAD

**Issue by rubenrouwhof** [MinBZK/amt#104]
# Functional requirement
Depends on: https://github.com/orgs/MinBZK/projects/7/views/9?pane=issue&itemId=73000793
**Discovery** 
https://app.clickup.com/9015737202/v/dc/8cp2evj-1815/8cp2evj-1795

**Description**
In our current concept, when a task is marked as “Done”, the output is automatically updated in the AMT System Card for that project. Currently, this process occurs entirely on the backend, and there is no logical interface to view the information contained in the AMT System Card. 

So we want to develop a minimalistic interface to display the System Card details when a task is marked as “Done”. This feature aims to provide a clear and simple way to access the updated information without adding unnecessary complexity to the user interface.

**Design criteria**
- Show last updated at

**Acceptance criteria**
- Use NLDS components
      - Header
      - Table
      - Link
- Show system card in projects/project_name/system_card
- Unordered list with assessment and model cards on top of page 
- Assessment and model card open in new page
- Show more -> directs to includes in table
- First letter of value should be capital
- Remove '_'
- For nested items we always show the first list, if there are more add link to expand other items.

Nice to have
- Show urls as link

**Designs**

![Image](https://github.com/user-attachments/assets/e3906c31-1855-4e2e-8180-e5f9d33e3e8a)
![Image](https://github.com/user-attachments/assets/157da541-3d80-42d1-8c0b-3cf2a1495689)

![Image](https://github.com/user-attachments/assets/b3a9e056-7d3d-4204-bee0-2efbc30a780c)

# Technical requirement

- We assume we have a system card **(locally)** that we visualize
- We make a new page -> Projects/{project_id}/system_card/
- Also we make new pages for assessments / technical tests -> Projects/{project_id}/system_card/{assesment_card_name}/ and Projects/{project_id}/system_card/{model_card_name}/
- The assessment and model cards should be clickable
- We assume that we have visualization components (otherwise we need to make them -> new ticket) 
- Keep in mind that it should be responsive (screen size for example)
- Navigation is in this ticket https://github.com/orgs/MinBZK/projects/7/views/9?pane=issue&itemId=69655776
- Inspiration can be drawn from https://github.com/cylynx/verifyml
